lib/commands/HelloWorld.ts

```diff
@@ -49,24 +48,20 @@ export class HelloWorld implements HandleCommand {
     @MappedParameter(MappedParameters.SlackUserName)
     public slackUser: string;
 
-    public handle(ctx: HandlerContext): Promise<HandlerResult> {
+    public async handle(ctx: HandlerContext): Promise<HandlerResult> {
```

Comment:
I wonder if this is a good pattern to use `async` and then don't use `await` in the function body. WDYT?

Reply:
Here is why I switched around the type assertions in the tests https://palantir.github.io/tslint/rules/no-object-literal-type-assertion/

Reply:
Rather, I thought adding the no-object-literal-type-assertion was a good idea for source, so I needed to rearrange the type assertions under test to accommodate that.

Reply:
Makes sense, thanks @ddgenome